src/molecules/Notifications/Notifications.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useMemo } from 'react';
+
+import { NotificationCounter } from 'atoms/NotificationCounter';
+import { NotificationsDataProps } from 'context';
+
+import { createNotificationMessages } from './utils/createNotificationMessages';
+import { getNotificationSum } from './utils/getNotificationSum';
+import { NotificationsWrapper, DropDownWindowWrapper, NotificationTextItem, NotificationTextWrapper} from './styled';
+
+export interface INotificationsProps {
+  notifications: NotificationsDataProps,
+}
+
+export interface NotificationsProps extends INotificationsProps {
```

Comment:
These interfaces have almost the same name. Rename one of them.
P.S. I see you IDE does not work properly...